scripts/statics/factsheets/area-stack.js

```diff
@@ -0,0 +1,151 @@
+import { select as d3_select } from 'd3-selection';
+import {
+    scaleLinear as d3_scale_linear,
+    scaleTime as d3_scale_time,
+    scaleOrdinal as d3_scale_ordinal,
+    schemeCategory10 as d3_schemeCategory_10
+  } from 'd3-scale';
+
+import {
+  area as d3_area,
+  stack as d3_stack
+} from 'd3-shape';
+
+import {
+  axisBottom as d3_axis_bottom,
+  axisLeft as d3_axis_left
+} from 'd3-axis';
+
+import { json as d3_json } from 'd3-request';
+import { extent as d3_extent } from 'd3-array';
```

Comment:
You managed to find all D3 imports. Congrats, you earned this GIF:

<img src="http://66.media.tumblr.com/54e8d7f01d0f11c3f0026182220dac4f/tumblr_n34jnhaTBX1s02vreo1_400.gif">